pymtl3/passes/rtlir/behavioral/test/BehavioralRTLIRL1Pass_test.py

```diff
@@ -724,7 +722,7 @@ class A( Component ):
     def construct( s ):
       @s.update
       def upblk():
-        class c(object): pass
+        class c: pass
```

Comment:
This looke like a slow kind of no-op - why not use a `pass` statement instead?

Reply:
This test case is trying to verify that the framework indeed catches a class definition inside the update block and throws a desired error message at it.